engine/src/main/java/io/camunda/zeebe/engine/state/mutable/MutableDecisionState.java

```diff
@@ -35,4 +35,6 @@ public interface MutableDecisionState extends DecisionState {
    * @param record the record of the decision
    */
   void deleteDecision(DecisionRecord record);
+
```

Comment:
:wrench: Should we also add some Javadocs here, like in the methods above?

Maybe something like the following:
```suggestion
/**
   * Deletes a decision requirements (DRG) from the state. Updates the latest version of the decision requirements (DRG) if the deleted
   * version is the latest version and a previous version is available.
   *
   * @param record the record of the decision requirements (DRG)
   */
```

Reply:
Oops, definitely. Nice catch 👍